pkg/csource/csource.go

```diff
@@ -445,19 +445,24 @@ func (ctx *context) hoistIncludes(result []byte) []byte {
 	}
 	result = includeRe.ReplaceAll(result, nil)
 	// Certain linux and bsd headers are broken and go to the bottom.
-	var sorted, sortedBottom []string
+	var sorted, sortedBottom, sortedTop []string
 	for include := range includes {
 		if strings.Contains(include, "<linux/") {
 			sortedBottom = append(sortedBottom, include)
 		} else if strings.Contains(include, "<netinet/if_ether.h>") {
 			sortedBottom = append(sortedBottom, include)
+		} else if ctx.target.OS == "freebsd" && strings.Contains(include, "<sys/types.h>") {
```

Comment:
Static checker does not like that "freebsd" string is duplicated too many times:
https://travis-ci.org/google/syzkaller/jobs/502958396#L2146
```
env CGO_ENABLED=1 GOMAXPROCS=1 GOGC=50 gometalinter.v2 ./...
pkg/csource/csource.go:454:30:warning: 2 other occurrence(s) of "freebsd" found in: pkg/csource/options.go:105:50 pkg/csource/options.go:121:64 (goconst)
pkg/csource/options.go:105:50:warning: 2 other occurrence(s) of "freebsd" found in: pkg/csource/csource.go:454:30 pkg/csource/options.go:121:64 (goconst)
pkg/csource/options.go:121:64:warning: 2 other occurrence(s) of "freebsd" found in: pkg/csource/csource.go:454:30 pkg/csource/options.go:105:50 (goconst)
```

Please add `const freebsd = "freebsd"` somewhere at the top and use it.

Reply:
Fixed.